crane/app.py

```diff
@@ -20,13 +29,16 @@ def create_app():
     """
     init_logging()
 
-    app = Flask(__name__)
+    app = CraneFlask(__name__)
     app.register_blueprint(v1.section)
     app.register_blueprint(v2.section)
     app.register_blueprint(crane.section)
     app.register_error_handler(exceptions.HTTPError, app_util.http_error_handler)
 
     config.load(app)
+    # to test send_file with flask's built in web server you need to set the following to 'false'
+    app.config['USE_X_SENDFILE'] = True
```

Comment:
Yes, we can do that. Since we will have four options related to "serve_content" then, I think it makes sense to put them into a new section. Defaults will be:

```
[serve_content]
enable: false
content_dir_v1: /var/www/pub/docker/v1/web/
content_dir_v2: /var/www/pub/docker/v2/web/
use_x_sendfile: true
```

The directory option defaults use the default web publisher locations. Thus, configuration for the simplest case (Crane runs on the same server as Pulp) will be:

```
[general]
...
data_dir: /var/www/pub/docker/

[serve_content]
enable: true
```